src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserPostDao.java

```diff
@@ -14,7 +14,7 @@
 @Slf4j
 public class JdbcUserPostDao implements UserPostDao {
 
-    private static final String INSERT = "INSERT INTO public.user_posts" +
+    private static final String INSERT_USER_MESSAGE = "INSERT INTO user_posts" +
```

Comment:
Can we rename INSERT_USER_MESSAGE to INSERT_USER_POST? 
Because everywhere POST used: (JdbcUserPostDao, INSERT INTO user_posts.....)

Reply:
+